tqdm/_tqdm.py

```diff
@@ -944,8 +945,10 @@ def set_postfix(self, **kwargs):
             # Else for any other type, try to get the string conversion
             elif not isinstance(kwargs[key], basestring):
                 kwargs[key] = str(kwargs[key])
+            # Else if it's a string, don't need to preprocess anything
```

Comment:
I call this the "ghost else" (I made it up): it often happens that else conditions are not explicitly set but implicitly, but the other conditions are designed with this implicit "ghost else" in mind. So in these cases, I add a comment to explain what the "ghost else" is meant to do.